KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyView.swift

```diff
@@ -124,6 +130,13 @@ final class KarhooLoyaltyView: UIStackView {
         return label
     }()
     
+    private lazy var balanceView: KarhooLoyaltyBalanceView = {
+        let view = KarhooLoyaltyBalanceView()
+        view.set(balance: presenter?.getBalance() ?? 0)
+        view.set(mode: .success)
+        return view
+    }()
+    
     // MARK: - Init
     init() {
```

Comment:
Performing low-level (sort of) setup within lifecycle methods is considered as bad practice, so I suggest, to extract all content of those methods to new methods.

Reply:
I've been researching where the best place to move the setup would be and found that the majority set up the UI just like I did here (e.g.: [HackingWithSwift](https://www.hackingwithswift.com/example-code/uikit/how-to-make-your-user-interface-in-code?fbclid=IwAR0UvMSEvJOWEqoboZe9KznT_h69Z326kveOt56sSKyjr-gk-AK6KlUpDKw))

Do you have any suggestion where else would be better to setup the view?

Reply:
Sorry, for some reason I thought there is no `setupView` method calling and all its content is called directly in initializer. Your solution is great, I have no comments 👍